sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosException.java

```diff
@@ -551,6 +551,31 @@ void setRntbdPendingRequestQueueSize(int rntbdPendingRequestQueueSize) {
                 public CosmosException createCosmosException(int statusCode, Exception innerException) {
                     return new CosmosException(statusCode, innerException);
                 }
+
+                @Override
+                public CosmosException createSerializableCosmosException(CosmosException cosmosException) {
+                    if (cosmosException == null) {
+                        return null;
+                    }
+                    CosmosException exception = new CosmosException(cosmosException.statusCode,
+                        cosmosException.cosmosError, cosmosException.getResponseHeaders());
+                    exception.requestTimeline = cosmosException.requestTimeline;
+                    exception.channelAcquisitionTimeline = cosmosException.channelAcquisitionTimeline;
+                    exception.rntbdChannelTaskQueueSize = cosmosException.rntbdChannelTaskQueueSize;
+                    exception.rntbdEndpointStatistics = cosmosException.rntbdEndpointStatistics;
+                    exception.lsn = cosmosException.lsn;
+                    exception.partitionKeyRangeId = cosmosException.partitionKeyRangeId;
+                    exception.requestUri = cosmosException.requestUri;
```

Comment:
Is this String or Uri? If the later, we should rather clone or String it.

Reply:
Yes, that's what I am thinking of doing, not in this iteration though, but the next one - when we create the model classes for these.
So added the TODO for now :)

Reply:
On second thoughts, will convert this in this PR itself :)

Reply:
I checked, and this is actually never set, so will leave it as it is for now. 
I think this can be safely removed in later iteration. For now, I won't remove it for the hotfix.